index.css

```diff
@@ -1,4 +1,127 @@
 * {
     box-sizing: border-box;
+    list-style: none;
 }
 
+body {
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.wrapper {
+    overflow: auto;
+    width: 100vw;
+    height: 100vh;
+}
+
+section {
+    float: left;
+    height: 100%;
+    width: 50%;
+}
+section div.table, section div.images {
+    width: 90%;
+    height: 80%;
+    margin: 10% 5%;
+}
+.table {
+    border:solid black;
+    height: 100%;
+    overflow-y: scroll;
+}
+
+.row {
+    width: 100%;
+}
+
+ul {
+    padding-top: 3%;
+    border-top: solid black;
+    width: 100%;
+    text-align: center;
+    overflow: auto;
+}
+
+.heading {
+}
```

Comment:
empty class - could be useful to add some css to distinguish heading from content